src/components/BotMessageFeedback.tsx

```diff
@@ -116,6 +117,7 @@ function BotMessageFeedback({ message }: { message: CoreMessageType }) {
           message?.myFeedback?.rating &&
           showFeedbackModal && (
             <MessageFeedbackModal
+              rootElementId={elementIds.widgetWindow}
```

Comment:
does this property exsit in MessageFeedbackModalProps?

Reply:
@bang9 `sendbird-modal__content` is being overwritten by widget.  It sets
  `max-width: calc(100% - 80px);` which is required for `MessageFeedbackFailedModal` but not for `MessageFeedbackModal`.

Reply:
> does this property exsit in MessageFeedbackModalProps?

No, I've added this to [uikit](https://github.com/sendbird/sendbird-uikit-react/compare/37951504a629bf4efd1f2903dcb5d42dc0a800a6...8f26cda305b87dd2b8a3fcfc558488d42d2bd485#diff-7b6184c31c6596319f7a88c49300984b3c42b635b03648df23dbd2af19aded88)

Reply:
the original implementation did not have 40px. it was smaller so the buttons were not overlaping like above. I [asked designer for answer](https://sendbird.slack.com/archives/C0772Q0BB8Q/p1719833180010909)